base/math.jl

```diff
@@ -91,7 +91,9 @@ deg2rad(z::Integer) = deg2rad(float(z))
 @vectorize_1arg Real rad2deg
 @vectorize_1arg Real deg2rad
 
-log(b,x) = log(x)./log(b)
+log{T<:Number}(b::T, x::T) = log(x)/log(b)
+log(b::Number, x::Number) = log(promote(b,x)...)
+@vectorize_2arg Number log
```

Comment:
This seems really small in the scheme of test time, almost anything that adds less than 10-100 seconds to our current time is probably worth the better coverage. Consider it a motivation to make codegen faster.